lib/Partitioner/Partitioner.cpp

```diff
@@ -485,14 +520,125 @@ NodeToFunctionMap Partitioner::selectPartitions(Function *F,
   // Step 2 : adjust the partition based on performance.
   partitionsAdjust(mapping, availableMemory);
 
+  // Step 3 : assign each partition with a logical device id. The partitions
+  // with the same logical device id will be assigned into the same physical
+  // device.
+  // logicalDeviceID_ = assignLogicalDeviceID(mapping, logicalDeviceID_);
```

Comment:
Why is this line commented out? Does it need to be removed?

Reply:
Yes, these lines (line 535-526) need to be removed.